packages/grafana-data/src/types/scopes.ts

```diff
@@ -83,3 +83,21 @@ export interface ScopeNode {
   };
   spec: ScopeNodeSpec;
 }
+
+export interface ScopeNavigationSpec {
```

Comment:
These shouldn' t live in grafana/data

Reply:
I wonder even if we move things to enterprise, if any enterprise plugin wants to interact with this they would need these types so maybe it makes sense to keep them here?

Reply:
That is the thing, there is nothing preventing any plugin or our code to use any of the API or clients they can access (so enterprise plugins can use enterprise API and clients). That is why we have a discussion about moving all the generated clients in a single place so the access to them is easy, instead of keeping them in "their" feature module.

Reply:
Can we make sure the generated types end up in `@grafana/data`, or do we have to do a bit of copy and paste?